index.d.ts

```diff
@@ -1,10 +1,10 @@
 import {
   APIGatewayEvent,
   APIGatewayEventRequestContext,
```

Comment:
Might be good to remove this since it's now unused

Reply:
It actually is used, the requestContext parameter in the `Request`  type is indeed an instance of this class.

Here's a snapshot from a production run of the value in that param: https://puu.sh/C56vJ/558c336edf.png

Reply:
Yep my bad again (getting late hehe 😅), checking out your fork now